### PR TITLE
fix: public schedule button disappearing

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6636

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/routes/public/sessions.js [hunk 1]
```diff
@@ -130,8 +130,16 @@ export default class SessionsRoute extends Route {
...
-      session : await this.infinity.model('sessions', {
+      event       : eventDetails,
+      dateSession : await this.infinity.model('sessions', {
+        sort         : params.sort || 'starts-at',
+        perPage      : 6,
+        startingPage : 1,
+        perPageParam : 'page[size]',
+        pageParam    : 'page[number]',
+        store        : eventDetails
+      }),
```
Comment (iamareebjamal) [inline app/routes/public/sessions.js:141]: Why?

Comment (maze-runnar) [reply app/routes/public/sessions.js:141]: when the session filter changes then the date also changes related to that filter. And on that date filter only that is gonna available.

Comment (iamareebjamal) [reply app/routes/public/sessions.js:141]: Sounds like a workaround

Comment (maze-runnar) [reply app/routes/public/sessions.js:141]: no, it's not a work around.

Comment (iamareebjamal) [reply app/routes/public/sessions.js:141]: Once the dates are set, make it so they don't change again

Comment (maze-runnar) [reply app/routes/public/sessions.js:141]: I have been trying it till now, but couldn't find another way.

**Comment by iamareebjamal** [comment]
Superseded by #6650

**Pull request closed without merging**